client/src/components/QueryComponent/QueryComponent.js

```diff
@@ -29,13 +30,17 @@ class QueryComponent extends Component {
     this.state = {
       queries: [],
       selectedQuery: defaultQuery,
+      selectedQueryMultiline: "",
       selectedQueryKey: null,
       queryKeyToBeRemoved: "",
       showSaveQueryModal: false,
       showConfirmDeleteModal: false,
       showConfirmOverwriteModal: false,
       newQueryName: "",
-      isOverlayResultsChecked: false
+      isOverlayResultsChecked: false,
+      rowCount: "20px",
+      display: false,
```

Comment:
Similar comment to above. Perhaps display is a little generic as a name?

Reply:
This Comment was addressed on Friday PR